docker/build.Dockerfile

```diff
@@ -87,12 +89,8 @@ RUN  CXX_COMPILER="g++" \
     && cmake --build . --target check \
     && cmake --build . --target install
 
-# Enable perfomance layers and test with `vkcube`.
-RUN export LD_LIBRARY_PATH=/performance-layers/build:$LD_LIBRARY_PATH \
-    && export VK_INSTANCE_LAYERS=VK_LAYER_STADIA_pipeline_compile_time \
-    && export VK_INSTANCE_LAYERS=$VK_INSTANCE_LAYERS:VK_LAYER_STADIA_pipeline_runtime \
-    && export VK_INSTANCE_LAYERS=$VK_INSTANCE_LAYERS:VK_LAYER_STADIA_pipeline_cache_sideload \
-    && export VK_INSTANCE_LAYERS=$VK_INSTANCE_LAYERS:VK_LAYER_STADIA_memory_usage \
-    && export VK_INSTANCE_LAYERS=$VK_INSTANCE_LAYERS:VK_LAYER_STADIA_frame_time \
-    && export VK_LAYER_PATH=/performance-layers/layer \
-    && xvfb-run vkcube --c 100 && echo Done!
+# Enable and test the performance layers.
+WORKDIR /performance-layers/docker
+RUN chmod u+x test_performance_layers.sh \ 
```

Comment:
Can't we change the permissions in the repo instead? You can see how the llpc CI does it: https://github.com/GPUOpen-Drivers/llpc/blob/dev/docker/llpc.Dockerfile#L37